### PR TITLE
Updating the Weekly Calendar

### LINKED ISSUE #5: Create Week Module

### LINKED ISSUE #6: Create Calendar Module

### PR DESCRIPTION
**Pull request by nheyland**
Just creating a PR for this if anyone has any input please. Ill let y'all know when I am ready for a review.

### CODE REVIEW

File: planner/views.py [hunk 1]
```diff
@@ -17,16 +17,15 @@ def get_date(req_day):
...
-    }
-    # Month
     d = get_date(request.GET.get('day', None))
     cal = Calendar(d.year, d.month)
     cal.setfirstweekday(6)
-    context['cal'] = cal.whole_month(withyear=True)
-    context['week'] = cal.whole_week(d.day)
-
+    context = {
+        'events': Event.objects.all(),
```
Comment (ewink1977) [inline planner/views.py:19]: Do you want to show all events here or only the user's events? If the latter, you will need to change this to `Event.objects.filter(created_by = current_user)` (a variable you probably want to set just below line 19 with `current_user = User.objects.get(id = request.session[user_id])`.

**Review by ewink1977 — CHANGES_REQUESTED** [review]
Made a single note, and it may be irrelevant, but otherwise, all looks good to me.

File: planner/views.py (hunk 1 above)
Comment (nheyland) [reply planner/views.py:19]: Yes, once I have the formatting down Ill be able to set it to just the users events, in the meantime keeping it like this so I have more to work with

**Comment by nheyland** [comment]
@ewink1977 @jarthurj @Jacqui-E-McGowan  Ready for functional review, will cleanup the code when I can though:

-Users must NOT be scheduling two things at the same time, we will need a validation for this. 
-Ill add the user filtering soon
-CSS for this is going to need some work
-Functional for now though 🤷🏼‍♀️

**Review by ewink1977 — APPROVED** [review]

**Pull request merged**